CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # [develop](https://github.com/adhearsion/electric_slide)
+  * Added `ElectricSlide::CallQueue#update_agent` to safely update a queued agent object's attributes
```

Comment:
@bklang Hmm, so is it `#add_agent` to add and `#remove_agent` then `#add_agent` to update? Or manually update the agent's attributes then `#return_agent` to update? I was trying to avoid complexity in maintaining the integrity of the queue. What if during an update, latter `#add_agent` or `#return_agent` steps fail? Is there an easy way to return the agent and the queue to the sane pre-update state (that's also not prone to race conditions)?